### PR TITLE
Release 2.0.5.rc0

### PR DESCRIPTION
**Pull request by JohanMabille**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by JohanMabille** [comment]
@conda-forge-admin please rerender

**Comment by jjerphan** [comment]
Oupsi

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+{% set release = "2024.12.09" %}
 
 package:
   name: mamba-split
   version: {{ mamba_version }}
 
 source:
   url: https://github.com/mamba-org/mamba/archive/refs/tags/{{ release }}.tar.gz
-  sha256: a86f37d140bb4bc7bf20e64346b9bcff2b33de4cbb969de86983e3b8a0775ab0
+  sha256: d5558cd419c8d46bdc958064cb97f963d1ea793866414c025906ec15033512ed
```
Comment (jjerphan) [inline recipe/meta.yaml:13]:
```suggestion
  sha256: 6a42fa12622b36448e5df1c1aa73158f23d41f2ed161b64012e8aed49f598917
```

**Review by jjerphan — COMMENTED** [review]
Let's call it a day?

**Pull request merged**